catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1644,6 +1685,232 @@ public void setMetacardTypes(List<MetacardType> types) {
         this.metacardTypes = types;
     }
 
+    @Override
+    public CreateResponse create(CreateRequest createRequest) throws IngestException {
```

Comment:
Oops, didn't look at that close enough. It still should be checked on the endpoint and handled. Do we gain much of anything by checking it here and throwing a runtime exception? Seems a little dangerous to throw a runtime exception.